test/test_helper.rb

```diff
@@ -15,7 +15,7 @@
 require "mocha/minitest"
 
 require "webmock/minitest"
-WebMock.disable_net_connect!
+WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
I notice in this commit `https://github.com/alphagov/travel-advice-publisher/pull/919/commits/bd74aa591b5e085d1436c51b0ef69773f3f19579` that the webdrivers gem is explicitly set to allow new Chromedrivers to be downloaded. Perhaps we could do something similar?